guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -44,107 +44,68 @@ This prevents the upgrade process from overwriting your DNS and DHCP configurati
 ----
 # rm /etc/yum.repos.d/*
 ----
-. Obtain the latest ISO files by following the {InstallingServerDisconnectedDocURL}downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in _{InstallingServerDisconnectedDocTitle}_.
 
-. Create directories to serve as a mount point, mount the ISO images, and configure the `rhel8` repository by following the {InstallingServerDisconnectedDocURL}configuring-the-base-operating-system-with-offline-repositories_satellite[Configuring the base operating system with offline repositories] procedure in _{InstallingServerDisconnectedDocTitle}_.
+. Export the required repositories from your connected {ProjectServer} to your disconnected {ProjectServer}:
+.. Ensure that the following repositories are synchronized on your connected server:
 +
-Do not install or update any packages at this stage.
-
-. Configure the {Project} {ProjectVersion} repository from the ISO file.
-
-.. Copy the ISO file's repository data file for the {ProjectName} packages:
-+
-[options="nowrap"]
-----
-# cp /media/sat6/Satellite/media.repo /etc/yum.repos.d/satellite.repo
-----
-
-.. Edit the `/etc/yum.repos.d/satellite.repo` file:
-+
-----
-# vi /etc/yum.repos.d/satellite.repo
-----
-
-... Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-{ProjectVersion}]
-----
-
-... Add the `baseurl` directive:
-+
-[options="nowrap"]
-----
-baseurl=file:///media/sat6/Satellite
-----
-
-. Configure the {ProjectName} Maintenance repository from the ISO file.
-
-.. Copy the ISO file's repository data file for {ProjectName} Maintenance packages:
-+
-[options="nowrap"]
-----
-# cp /media/sat6/Maintenance/media.repo /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-.. Edit the `/etc/yum.repos.d/satellite-maintenance.repo` file:
-+
-[options="nowrap"]
-----
-# vi /etc/yum.repos.d/satellite-maintenance.repo
-----
-
-... Change the default `InstallMedia` repository name to `{Project}-Maintenance`:
+* {RepoRHEL9BaseOS}
+* {RepoRHEL9AppStream}
+* {RepoRHEL9ServerSatelliteServerProjectVersion}
+.. Export the repositories in a syncable format:
 +
-[options="nowrap" subs="+quotes,attributes"]
-----
-[{Project}-Maintenance]
-----
-
-.. Add the `baseurl` directive:
-+
-[options="nowrap"]
+[options="nowrap" subs="+quotes,verbatim,attributes"]
 ----
-baseurl=file:///media/sat6/Maintenance/
+$ hammer content-export complete repository \
+--id=_Repo_ID_ --format=syncable
```

Comment:
To export the repository, users must have synchronized it before with download policy "immediate".

Also, I suggest to add a command to get the Repo IDs and explain that you need to run this command three times: once for each repo listed in lines 51-53.